Assets/Plugins/LeapMotion/Internal/VRVisualizer/ScenesResources/Scripts/VisualizerManager.cs

```diff
@@ -62,7 +61,7 @@ private void goScreenTop()
         m_PCVisualizer.gameObject.SetActive(true);
         m_VRVisualizer.gameObject.SetActive(false);
         m_warningText.text = "ScreenTop tracking mode activated";
-        m_controller.SetTrackingMode(eLeapTrackingMode.eLeapTrackingMode_ScreenTop);
+        m_controller.SetPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_SCREENTOP);
```

Comment:
It's a bit strange that we set the policy flag here for screentop, when we don't set the flag for HMD in goVR